src/osx/carbon/utilscocoa.mm

```diff
@@ -555,6 +555,8 @@ wxString wxStringWithNSString(NSString *nsstring)
         formerAppearance = NSAppearance.currentAppearance;
         NSAppearance.currentAppearance = NSApp.effectiveAppearance;
     }
+#else
+    (void)formerAppearance;
```

Comment:
This is extremely minor, but we have `wxUnusedVar()` which is a more readable way of doing the same thing.